datafusion/common/src/config.rs

```diff
@@ -270,7 +270,48 @@ config_namespace! {
         /// will be reordered heuristically to minimize the cost of evaluation. If false,
         /// the filters are applied in the same order as written in the query
         pub reorder_filters: bool, default = false
+
+        // The following map to parquet::file::properties::WriterProperties
+
+        /// Sets best effort maximum size of data page in bytes
+        pub data_pagesize_limit: usize, default = 1024 * 1024
+
+        /// Sets best effort maximum number of rows in data page
+        pub data_page_row_count_limit: usize, default = usize::MAX
+
+        /// Sets best effort maximum dictionary page size, in bytes
+        pub dictionary_page_size_limit: usize, default = 1024 * 1024
+
+        /// Sets maximum number of rows in a row group
+        pub max_row_group_size: usize, default = 1024 * 1024
+
+        /// Sets "created by" property
+        pub created_by: String, default = concat!("parquet-rs version ", env!("CARGO_PKG_VERSION")).into()
+
+        pub compression: Option<String>, default = None
+
+        /// Sets default encoding for any column
+        pub encoding: Option<String>, default = None
+
+        /// Sets if dictionary encoding is enabled
+        pub dictionary_enabled: Option<bool>, default = None
+
+        /// Sets if statistics are enabled for any column
+        pub statistics_enabled: Option<String>, default = None
+
+        /// Sets max statistics size for any column
+        pub max_statistics_size: Option<usize>, default = None
+
+        /// Sets if bloom filter is enabled for any column
+        pub bloom_filter_enabled: Option<bool>, default = None
     }
+        // TODO macro not working with Option<f64> or Option<u64>
```

Comment:
👍  that would be cool to clean up if possible prior to a real PR -- let me know if you need some help

Reply:
This is fixed! It was as simple as adding:

```rust
config_field!(f64);
config_field!(u64);
```